official/vision/dataloaders/retinanet_input.py

```diff
@@ -39,17 +39,17 @@ class Parser(parser.Parser):
 
   def __init__(self,
                output_size,
-               min_level: int | None,
                max_level,
-               num_scales: int | None,
-               aspect_ratios: list[float] | None,
-               anchor_size: float | None,
+               min_level: Optional[int] = None,
```

Comment:
Can you keep the original order of arguments?

Reply:
Updated the suggested changes.